compass/ocean/tests/utility/combine_topo/__init__.py

```diff
@@ -158,169 +180,410 @@ def _modify_bedmachine(self):
+        out_filename : `str`, weights file name
+        """
+        config = self.config
+        method = config.get('combine_topo', 'method')
+
+        # Generate weights file
+        args = [
+            'ESMF_RegridWeightGen',
+            '--source', in_filename,
+            '--destination', self.outputs[0],
+            '--weight', out_filename,
+            '--method', method,
+            '--netcdf4',
+            '--src_regional',
+            '--ignore_unmapped',
+        ]
+        run_command(
+            args, self.cpus_per_task, self.ntasks,
+            self.openmp_threads, config, self.logger,
+        )
+
+    def _remap_to_target(
+        self, in_filename, mapping_filename, out_filename, default_dims=True,
+    ):
+        """
+        Remap to `self.target_grid`. Filenames are passed as parameters so
+        that the function can be applied to GEBCO and BedMachine.
 
-        in_descriptor = ProjectionGridDescriptor.create(
-            projection, x, y, 'BedMachineAntarctica_500m')
+        Parameters
+        ----------
+        in_filename : `str`, source file name
+        mapping_filename : `str`, weights file name
+        out_filename : `str`, remapped file name
+        default_dims : `bool`, default `True`,
+            if `False` specify non-default source dims y,x
+        """
+        config = self.config
+        section = config['combine_topo']
+        renorm_thresh = section.getfloat('renorm_thresh')
+
+        # Build command args
+        args = [
+            'ncremap',
+            '-m', mapping_filename,
+            '--vrb=1', f'--renormalize={renorm_thresh}',
+        ]
+
+        # Add non-default gridding args
+        regridArgs = []
+        if not default_dims:
+            regridArgs.extend([
+                '--rgr lat_nm=y',
+                '--rgr lon_nm=x',
+            ])
+        if self.target_grid == 'lat_lon':
+            regridArgs.extend([
+                '--rgr lat_nm_out=lat',
+                '--rgr lon_nm_out=lon',
+                '--rgr lat_dmn_nm=lat',
+                '--rgr lon_dmn_nm=lon',
+            ])
+        if len(regridArgs) > 0:
+            args.extend(['-R', ' '.join(regridArgs)])
+
+        # Append input and output file names
+        args.extend([in_filename, out_filename])
+
+        # Remap to target grid
+        check_call(args, self.logger)
+
+    def _remap_gebco(self):
+        """
+        Remap GEBCO to `self.target_grid`
+        """
+        logger = self.logger
+        logger.info('Remapping GEBCO data')
 
-        out_descriptor = LatLonGridDescriptor.read(fileName=gebco_filename)
+        # Parse config
+        config = self.config
+        section = config['combine_topo']
+        global_name = section.get('global_filename').strip('.nc')
+        method = section.get('method')
+        lat_tiles = section.getint('lat_tiles')
+        lon_tiles = section.getint('lon_tiles')
+
+        # Make tiles directory
+        os.mkdir('tiles')
```

Comment:
I think we want:
```suggestion
        os.makedirs('tiles', exist_ok=True)
```
This works if `tiles` already exists.  The alternative is that we want to delete `tiles` if it exists and start fresh.

Reply:
I don't think it matters which, since everything gets rerun either way.

Reply:
Happy with your changes, I did a hard reset to your fork.